index.js

```diff
@@ -2963,10 +2963,11 @@
 
   //. ### Array
 
-  //# append :: a -> Array a -> Array a
+  //# append :: (Applicative f, Semigroup (f a)) => a -> f a -> f a
   //.
-  //. Takes a value of any type and an array of values of that type, and
-  //. returns the result of appending the value to the array.
+  //. Takes a value of any type and a (Applicative, Semigroup) of values of
+  //. that type, and returns the result of appending the value to the
+  //. (Applicative, Semigroup).
```

Comment:
How about:

*Returns the result of appending the first argument to the second.*